.github/workflows/build.yml

```diff
@@ -21,16 +21,10 @@ jobs:
         fetch-depth: 0
         repository: ${{ github.event.pull_request.head.repo.full_name || github.repository }}
 
-    - name: Native Build
-      run: |
-        mkdir build && cd build
-        pip install gcovr
-        sudo bash ../install-prerequisites.sh
-        cmake -DCMAKE_BUILD_TYPE=DEBUG ..
-        make
-
     - name: Run unit test
-      run: cd build && make coverage
+      run: |
+        docker run --name jasminegraph-prerequisite -v $(pwd):/home/ubuntu/software/jasminegraph -d -t -i miyurud/jasminegraph /bin/bash
```

Comment:
Purpose of this?

Reply:
Need the jasminegraph-prerequisite image alive to `docker exec` and run the commands needed.
Refer: https://docs.docker.com/engine/reference/commandline/exec/#examples